ros_buildfarm/templates/devel/devel_job.xml.em

```diff
@@ -356,6 +356,18 @@ if pull_request:
     remove_prefix='collated_test_stats',
 ))@
 @[end if]@
+@[if (not pull_request) and publish_test_results]@
+@(SNIPPET(
+    'publisher_publish-over-ssh',
+    config_name='docs',
+    remote_directory='%s/devel_jobs/%s/test_results' % (rosdistro_name, source_repo_spec.name),
+    source_files=[
+        'ws/test_results/**'
+    ]
+    remove_prefix='ws/test_results',
+    clean_remote='true',
```

Comment:
Nitpick: I would suggest passing a Python boolean here `True` and updating the template accordingly.

Reply:
Thanks! Good point!
I changed it accordingly.